database/dtb.ksy

```diff
@@ -0,0 +1,128 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
+  application:
+    - Linux
+    - Das U-Boot
+  xref:
+    - wikidata: Q16960371
+  license: CC0-1.0
+  endian: be
+doc: |
+  Also referred to as Devicetree Blob (DTB). It is a flat
+  binary encoding of data (primarily devicetree data, although
+  other data is possible as well).
+
+  On Linux systems that support this the blobs can be accessed in
+  /sys/firmware/fdt
+
+  https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-firmware-ofw
```

Comment:
Is this stray URL related to the above or below paragraph? Can you make it clear?

Reply:
I made it a bit clearer (I hope). Please verify.

Reply:
Yes, looks better.